lix/packages/sdk/src/query-utilities/get-lowest-common-ancestor.ts

```diff
@@ -50,13 +60,26 @@ export async function getLowestCommonAncestor(args: {
 			.selectFrom("change")
 			.selectAll()
 			.where("id", "=", nextChange.id)
+			// TODO account for multiple parents
 			.executeTakeFirst();
 
 		if (changeExistsInTarget) {
 			return changeExistsInTarget;
 		}
 
-		parentId = nextChange.parent_id;
+		const parentEdges = await args.sourceLix.db
+			.selectFrom("change_edge")
+			.selectAll()
+			.where("child_id", "=", nextChange.id)
+			.execute();
+		// TODO assumes that only one parent exists
+		if (parentEdges.length > 1) {
+			// https://github.com/opral/inlang-sdk/issues/134
+			throw new Error(
+				"Unimplemented. Multiple parents not supported until the branch feature exists.",
+			);
+		}
```

Comment:
One of the TODOS that should be tackled after the branch milestone.

branching will influence the logic
working on this now is a waste of time